packages/key-management/src/util/mapHardwareSigningData.ts

```diff
@@ -163,7 +163,7 @@ const prepareTrezorOutputs = (
       const destination = ownAddress
         ? {
             addressParameters: {
-              addressType: trezor.CardanoAddressType.BASE,
+              addressType: trezor.PROTO.CardanoAddressType.BASE,
```

Comment:
Nitpick: just wondering if we should make `trezor` capitalized. We seem to apply this convention in SDK packages eg. `KeyManagement.TrezorKeyAgent.initializeTrezorTransport`.

Reply:
@marcin-mazurek I kept it as is since the `ledger` is also not capitalized (https://github.com/input-output-hk/cardano-js-sdk/blob/047aed18216d3b03abf7c06d0931015a40d10ee2/packages/key-management/src/util/mapHardwareSigningData.ts#L4). 
It is changed in new dedicated `hardware-trezor` package https://github.com/input-output-hk/cardano-js-sdk/blob/12d8422fadb3ced2d0ffc1d00c611d3728713870/packages/hardware-trezor/src/TrezorKeyAgent.ts#L3

Reply:
So `Trezor` used to be PascalCase before your changes. Lowercase `ledger` feels inconsistent with internal SDK types too:

![image](https://github.com/input-output-hk/cardano-js-sdk/assets/6684554/c4c275d6-3a6c-4028-99b0-ae456213082c)

Thoughts @AngelCastilloB / @mkazlauskas / @rhyslbw?

Reply:
Sorry - it was TrezorConnect that was pascal case. But still something I'd like to consider :)

Reply:
We might have some inconsistencies but in general yes, PascalCase is used for namespaces

Reply:
`ledger` is an import so doesn't impact the API, but yeah should probably be `Ledger`

Reply:
Ok, @tomislavhoracek explained to me on a call that this file will be removed, so it's totally fine to keep the diff to minimum :)